generators/generator-bot-calendar/generators/app/templates/README.md

```diff
@@ -10,25 +10,46 @@ In order to test this bot locally, you will need the following services provisio
 - Azure Bot Registration
 - Language Understanding (LUIS)
 
+#### Configure Microsoft App Password
```

Comment:
What do you think of giving people a high level topic/summary of all the steps needed at the beginning of this doc. This is just to prep people on basically at a high level what they should expect as far as setup is concerned. 

This way I think it can also prevent people from feeling they are stuck in "manual hell" where they just keep following steps with no end in sight when they are half way through?

Reply:
Added